std_msgs/Header header

uint8 state
uint8 activation_state
uint8 fault_state
uint8 go_to_state
uint8 requested_position_echo
uint8 finger_position
uint8 finger_current
uint8 obj_detection
uint8 gripper_open

string activation_state_msg
string fault_msg
string go_to_msg
string obj_detection_msg